Sources/Clappr/Classes/Plugin/Core/MediaControl/FullscreenButton.swift

```diff
@@ -0,0 +1,75 @@
+open class FullscreenButton: MediaControlPlugin {
+    private var icon = UIImage.fromName("fullscreen", for: FullscreenButton.self)
```

Comment:
The previous fullscreen button toggled between two icons. One displayed when in fullscreen and other when in windowed mode.

Can we reproduce the same behavior here?

Reply:
Yes, we can.

=]